index.md

```diff
@@ -301,78 +301,54 @@ should therefore not be put in a version control repository.
+4.  Avoid duplication and manual entry of information, particularly in constructing bibliographies, tables of contents, and lists.
+5.  Make it easy to regenerate the final shared form (e.g., the PDF), and to tell if the PDF in hand is up to date.
+6.  Make it easy to share the final version with collaborators and submit it to journals.
 
 ### Rules
 
-We would like to require this for papers, theses, technical reports, and other manuscripts:
+There are two alternative approaches we recommend:
 
-1.  Manuscripts are written in a plain text format such as LaTeX or Markdown that plays nicely with version control
-2.  Tools needed to compile manuscripts are managed just like tools used to do simulation or analysis
+####Distributed web-based system
 
-But to quote [Stephen Turner][turner-comment-docs]:
+This is technically relatively straightforward: 
 
-> ...try explain the notion of compiling a document to an overworked physician you collaborate with.
-> Oh, but before that, you have to explain the difference between plain text and word processing.
-> And text editors.
-> And markdown/LaTeX compilers.
-> And BiBTeX.
-> And Git.
-> And GitHub. Etc.
-> Meanwhile he/she is getting paged from the OR...
->
-> ...as much as we want to convince ourselves otherwise,
-> when you have to collaborate with those outside the scientific computing bubble,
-> the barrier to collaborating on papers in this framework is simply too high to overcome.
-> Good intentions aside,
-> it always comes down to "just give me a Word document with tracked changes," or similar.
-> There's always a least common denominator who just isn't going to be on board for writing like this.
-
-We therefore recommend:
-
-*   *Either:*
-    1.  Manuscripts are written in a plain text format such as LaTeX or Markdown that plays nicely with version control
-    2.  Tools needed to compile manuscripts are managed just like tools used to do simulation or analysis
-*   *Or:*
-    1.  Manuscripts are written using Google Docs or some other online tools with rich formatting and change tracking
-    2.  A short text file is added to the `doc` directory with metadata about each online manuscript
-        *   Just as the `data` directory might contain links rather than actual data
-    3.  The manuscript is downloaded and saved in `doc` in an editable form (e.g., `.docx` or `.odt`) after major changes
-*   Either way, use a distributed web-based system for managing the manuscript
-    so that the master document is clearly defined and everyone can collaborate on an equal footing
+1.  a manuscript is written using Google Docs or some other online tool with rich formatting, change tracking, and reference management.
+2. In the project folder, add a short text file to the `doc` directory with metadata about each online manuscript (e.g. the URL). This is analogous to the `data` directory, which might contain links to the location of the datafile(s), rather than the actual files.
```

Comment:
What should this file be called?  We're making specific recommendations elsewhere about filenames.